packages/server/api/controllers/tools.controller.js

```diff
@@ -0,0 +1,60 @@
+/* TODO: This is an example controller to illustrate a server side controller.
+Can be deleted as soon as the first real controller is added. */
+
+const knex = require('../../config/db');
+const HttpError = require('../lib/utils/http-error');
+
+const getTools = async () => {
+  return knex('tools').select(
+    'name',
+    'time_frame_min',
+    'time_frame_max',
+    'group_size_min',
+    'group_size_max',
+    'facilitation_level',
+    'materials',
+    'pitch',
+    'description',
+    'instructions',
+    'source',
+    'picture',
+    'created_at',
+  );
+};
+
+const getToolById = async (id) => {
+  if (!id) {
+    throw new HttpError('Id should be a number', 400);
```

Comment:
If you aim to validate that ID is a number, something like 
`!Number(id)
`would be more appropriate.